ktlint-cli/build.gradle.kts

```diff
@@ -107,18 +106,9 @@ val shadowJarExecutable by tasks.registering(DefaultTask::class) {
             appendBytes(ktlintCliAllJarFile.readBytes())
 
             setExecutable(true, false)
-
-            if (isReleaseBuild) {
-                logger.lifecycle("Creating the signature file: $selfExecutableKtlintSignatureOutputPath")
-                signing.sign(this)
```

Comment:
This was reason why configuration cache had been failing. Since `signing.sign()` is just a config, it doesn't have to be within `doLast` block.